x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/schedule.ts

```diff
@@ -955,11 +954,10 @@ export default function scheduleBackfillTests({ getService }: FtrProviderContext
             // User has read privileges in this space
             case 'global_read at space1':
               expect(response.statusCode).to.eql(403);
-              expect(response.body).to.eql({
-                error: 'Forbidden',
-                message: `Unauthorized by "alertsFixture" to scheduleBackfill "test.patternFiringAutoRecoverFalse" rule`,
-                statusCode: 403,
-              });
+              expect(response.body.error).to.eql('Forbidden');
+              expect(response.body.message).to.match(
+                /Unauthorized by "alertsFixture" to scheduleBackfill "[^"]+" rule/
```

Comment:
we're scheduling multiple rule types in one request so the error message would sometimes reference a different rule type id